assets/messages.yml

```diff
@@ -1039,6 +1039,19 @@ messages:
       shortcut: qul
       message: "%quick_links%"
       fillname: []
+  removed-version:
```

Comment:
For consistency the ID should possibly start with `panel-` since the complete message is inside a panel. (Even though its content is similar to `provide-affected-versions`)